src/Bluejay.asm

```diff
@@ -395,22 +394,14 @@ Eep_Pgm_LED_Control: DB DEFAULT_PGM_LED_CONTROL ; EEPROM copy of programmed LED
 Eep_Pgm_Power_Rating: DB DEFAULT_PGM_POWER_RATING ; EEPROM copy of programmed power rating
 
 Eep_Dummy: DB 0FFh                      ; EEPROM address for safety reason
-IF MCU_TYPE == 2
-    CSEG AT 3060h
-ELSE
-    CSEG AT 1A60h
-ENDIF
+CSEG AT CSEG_NAME
 Eep_Name: DB "Bluejay         "         ; Name tag (16 Bytes)
 
-IF MCU_TYPE == 2
-    CSEG AT 3070h
-ELSE
-    CSEG AT 1A70h
-ENDIF
+CSEG AT CSEG_MELODY
 Eep_Pgm_Beep_Melody: DB 2,58,4,32,52,66,13,0,69,45,13,0,52,66,13,0,78,39,211,0,69,45,208,25,52,25,0
 
     Interrupt_Table_Definition          ; SiLabs interrupts
-CSEG AT 80h                             ; Code segment after interrupt vectors
+CSEG AT CSEG_AFTER_INTERRUPTS           ; Code segment after interrupt vectors
```

Comment:
I would use CSEG_APP, because at this code segment is where APP code starts

Reply:
Done.